fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -317,7 +316,7 @@ public Void visitArithmeticExpr(ArithmeticExpr node, Scope scope) {
                                     + " is invalid.");
                 }
 
-                Function fn = Expr.getBuiltinFunction(op.getName(), new Type[] {commonType, commonType},
+                Function fn = Expr.getBuiltinFunction(op.getName(), new Type[]{commonType, commonType},
```

Comment:
Don't use the style

Reply:
Sorry, forget to import new coding style xml file.